docker/init.sql

```diff
@@ -7,6 +8,6 @@ CREATE DATABASE exercism_v3_dj_development;
 ALTER DATABASE exercism_v3_dj_development CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
 GRANT ALL PRIVILEGES ON exercism_v3_dj_development.* TO 'exercism_v3';
 
-CREATE DATABASE `exercism_v3_test`;
-ALTER DATABASE `exercism_v3_test` CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
-GRANT ALL PRIVILEGES ON `exercism_v3_test`.* TO 'exercism_v3';
+CREATE DATABASE exercism_v3_test;
+ALTER DATABASE exercism_v3_test CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+GRANT ALL PRIVILEGES ON exercism_v3_test.* TO 'exercism_v3';
```

Comment:
Why this change?

Reply:
Oh, consistency with the other statements in the file

Reply:
So, the background to this is that you can't have hyphens in db names unless they're surrounded with `. And the test databases get split one per processor, so get named `exercism_v3_test-1`. They used to be specified here but they're (correctly) not now. I think I'd actually rather go the other way and add them everywhere in that file, as it makes things safer to have them in case we ever rename things.

Reply:
Makes sense! Will do that